src/widgets/map/ui/mapInitializer.tsx

```diff
@@ -86,12 +147,49 @@ export const MapInitializer = () => {
     };
   }, [map, isMapIdle]);
 
+  // 해당 이펙트는 place가 ROUTER_PATH.PLACE 면서 state가 없을 때 실행됩니다.
+  // 이펙트의 용도는 placeParams의 값이 존재하지 않거나 boundsParams의 범위를 벗어난 경우에 placeParams 값을 boundsParams의 중심으로 설정하기 위함입니다.
+  useEffect(() => {
+    const { southWestLat, northEastLat, southWestLng, northEastLng } =
+      boundsParams;
+
+    if (!(southWestLat && northEastLat && southWestLng && northEastLng)) {
+      return;
+    }
+
+    if (ROUTER_PATH.PLACE === pathname && !state) {
+      const { lat, lng } = placeParams;
+
+      if (!(southWestLat && northEastLat && southWestLng && northEastLng)) {
+        return;
+      }
```

Comment:
이 조건문에서 lat과 lng이 없을 경우도 추가하는 건 어떨까요?
```suggestion
      if (!(southWestLat && northEastLat && southWestLng && northEastLng && lng && lat)) {
        return;
      }
```

Reply:
오히려 lat lng가 없는 경우에 시행되어야 하는 effect입니다

Reply:
밑의 코드 줄에서 그 경우를 다루고 있숩니다

lat, lng 가 있고 바운더리 내부에 있을 때로요

Reply:
아 그리고 lat , lng 가 없다면 설정해줘야 하기 때문에 오히려 lat  , lng 가 없다면 boundsParms 로 센터값 지정 , 있다면 유효성 검증하고 수정 or 사용 해야 합니다